### PR TITLE
Tests for Mappers 1-8

### PR DESCRIPTION
**Pull request by SezamUA**

### CODE REVIEW

**Review by DZotsenko — APPROVED** [review]

**Review by DwemOV — APPROVED** [review]

File: service/src/test/java/greencity/mapping/EcoNewsCommentVOMapperTest.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+import java.time.LocalDateTime;
+
+import java.util.HashSet;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class EcoNewsCommentVOMapperTest {
+    private EcoNewsCommentVOMapper mapper;
+    private LocalDateTime now;
+    EcoNewsComment parentEcoNewsComment;
```
Comment (maks741) [inline service/src/test/java/greencity/mapping/EcoNewsCommentVOMapperTest.java]:
```suggestion
    private EcoNewsComment parentEcoNewsComment;
```

File: service/src/test/java/greencity/mapping/EcoNewsCommentVOMapperTest.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    EcoNewsCommentVO expectedParentEcoNewsCommentVO;
```
Comment (maks741) [inline service/src/test/java/greencity/mapping/EcoNewsCommentVOMapperTest.java]:
```suggestion
    private EcoNewsCommentVO expectedParentEcoNewsCommentVO;
```

**Review by maks741 — APPROVED** [review]

**Review by StannyB9 — APPROVED** [review]

File: service/src/test/java/greencity/mapping/EcoNewsCommentVOMapperTest.java (hunk 1 above)
Comment (SezamUA) [reply service/src/test/java/greencity/mapping/EcoNewsCommentVOMapperTest.java]: Thank You for point out on this mistake. It was made during refactoring. I fixed it.

File: service/src/test/java/greencity/mapping/EcoNewsCommentVOMapperTest.java (hunk 2 above)
Comment (SezamUA) [reply service/src/test/java/greencity/mapping/EcoNewsCommentVOMapperTest.java]: Thank You for point out on this mistake. It was made during refactoring. I fixed it.

**Pull request merged**